codeU1.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri May 19 15:10:45 2017
+
+@author: Luiza
+"""
+
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 18 10:12:38 2017
+CodeU exercise 1
+@author: Luiza
+"""
+
+# Task 1: Given 2 strings, decide if one of them is a permutation of the other
+# Return:

```

Comment:
Documenting the behavior is nice - makes it easier to use the library and serves as a lightweight contract between the library and users.